frontend/src/app/app.module.ts

```diff
@@ -184,9 +191,8 @@ export const MY_FORMATS = {
       useClass: CustomRouter,
     },
     TranslateService,
+    provideHttpClient(withInterceptorsFromDi()),
```

Comment:
Do we need `withInterceptorsFromDi()`?

The docs state: `Prefer withInterceptors and functional interceptors instead, as support for DI-provided interceptors may be phased out in a later release`

Reply:
Is still supported in next Angular version: https://next.angular.dev/api/common/http/withInterceptorsFromDi?tab=api

I want to use Dependency Injection for HTTP Interceptors due to better Overview of interceptor service.